code/src/java/pcgen/util/SkinLFResourceChecker.java

```diff
@@ -46,31 +37,16 @@ private SkinLFResourceChecker()
 	 * Get the missing resource count
 	 * @return the missing resource count
 	 */
-	public static int getMissingResourceCount()
+	public static boolean hasMissingResources()
 	{
 		return missingResourceCount;
 	}
 
-	/**
-	 * Get the missing resource message
-	 * @return the missing resource message
-	 */
-	public static String getMissingResourceMessage()
-	{
-		if (missingResourceCount != 0)
-		{
-			return resourceBuffer + "\n" + ResourceChecker.GET_IT_HERE_MSG + WHERE_TO_GET_IT + '\n'
-				+ ResourceChecker.MISSING_LIB_MSG; //TODO Why does this have hardcoded file separators? JK070115
-		}
-
-		return "";
-	}
-
 	private static void checkResource()
 	{
 		if (!ResourceChecker.hasResource("com.l2fprod.gui.plaf.skin.SkinLookAndFeel", "skinlf.jar", resourceBuffer))
 		{
-			++missingResourceCount;
+			missingResourceCount = true;
```

Comment:
Terminology matters to me. Why are we calling a Boolean a "Count"?  Count means a value of an Integer or at the very least a number. I'm happy you're reusing and refactoring the code, but can we rename the var to match its new purpose?  Above the function call is hasMissingResources, why not name the Boolean hasMissingResource which matches name to intent.